mk/uvmt/vsim.mk

```diff
@@ -150,9 +150,10 @@ VSIM_SCRIPT_DIR	   = $(abspath $(MAKE_PATH)/../tools/vsim)
 
 VSIM_UVM_ARGS      = +incdir+$(UVM_HOME)/src $(UVM_HOME)/src/uvm_pkg.sv
 
+VSIM_FLAGS += -sv_lib $(basename $(abspath $(IMPERAS_DV_MODEL)))
```

Comment:
This line was just move.
And the move is newer (2 months) https://github.com/openhwgroup/core-v-verif/blame/7fdba5f68d5344712e4bda5cb5297734f909ab4e/mk/uvmt/vsim.mk#L153
...than the non-moved one (3 months) https://github.com/openhwgroup/core-v-verif/blame/9b184c294e609660a2808b0b101dc1d90891b995/mk/uvmt/vsim.mk#L167